docs/general/polkadot-uc.md

```diff
@@ -0,0 +1,337 @@
+core to access secure, decentralized, and ubiquitous computation.
+
+## From Slot Auctions to Coretime Marketplace
+
+The end product of blockchains is [**Blockspace**](./polkadot-v1.md#polkadots-blockspace).
+Applications need to access Polkadot's blockspace, and the entry points to blockspace are the cores.
+Thus, applications will need to reserve some time on cores or **Coretime** to gain the right to
+access Polkadot's secure blockspace and interoperability for a finite period.
+
+Cores are agile and general: they can change what job they run as easily as a modern CPU. It follows
+that the procurement of those cores must be agile as well. The current slot auction mechanism is not
+agile, creates high entry barriers, and is designed for long-running single applications (i.e., the
+original Polkadot vision proposed in the whitepaper).
+
+We depart from the classic lease auctions and propose an agile marketplace for coretime where two
+Polkadot-native sale options of coretime are possible: **bulk** and **instantaneous**. This setup
+maximizes the agility of Polkadot and lets the market figure out the best solution needed for the
+applications to be successful.
+
+### Bulk Coretime Rental
+
+Bulk coretime rental (primary market) is a monthly sale of 4 weeks of coretime at a fixed price with
+a target of, e.g., 75% of the available cores. The price will change depending on the deviation from
+the target value, and unrented cores go to the instantaneous market. Special considerations might be
+necessary for pre-existing applications.
+
+Bulk coretime does not need to be directly deployed because it is an asset, more precisely, a
+non-fungible asset (unless it is for the same period). Coretime tokens are sold in a 4-week period
+and can be split into multiple NFTs of smaller periods. Coretime NFTs are sold and assigned by a
+broker system parachain to the applications now owning the NFTs, which must be consumed to allocate
+computation on a Polkadot core. The broker parachain records the price and the assignment, and the
+coretime can always be purchased the following month for the same assignment (i.e., same
+application) with a capped price change. This rent control allows long-running applications to have
+certainty and predictability about coretime costs (not possible with lease auctions). The process of
+assigning coretime is irreversible.
+
+In general, bulk coretime NFTs can be:
+
+- sold to one application (similar to the lease auctions in Polkadot 1.0, with the difference that
+  coretime in Polkadot 1.0 is not sold, but applications need to lock their DOT tokens)
+- sold to several applications taking turns on a core
+- placed on the instantaneous market
+- exposed and sold by a broker system parachains via [XCM](./learn-xcm-index) NFT interface to
+  Polkadot-powered applications that want to allow for these NFTs to be exchanged and traded on
+  their market.
```

Comment:
this is a lot of specific implementation details rather than simply a vision / direction.

Reply:
Agree, I will propose a solution for this

Reply:
I deleted the part about bulk and instantaneous coretime rental